torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -64,6 +58,73 @@
     "get_x86_inductor_linear_dynamic_fp16_config",
 ]
 
+# In the absence of better name, just winging it with QuantizationConfig
+@dataclass(eq=True, frozen=True)
+class QuantizationConfig:
```

Comment:
we can probably put these in quantizer/utils.py since these are shared, but should be clear that there is no requirement of quantizers using them

Reply:
I can move them, but IIRC no one else is using it so kept it here.